easybuild/easyconfigs/p/pytest-rerunfailures/pytest-rerunfailures-12.0-GCCcore-12.3.0.eb

```
@@ -0,0 +1,24 @@
+easyblock = 'PythonPackage'
+
+name = 'pytest-rerunfailures'
+version = '12.0'
+
+homepage = 'https://github.com/pytest-dev/pytest-rerunfailures'
+description = """pytest plugin to re-run tests to eliminate flaky failures."""
+
+toolchain = {'name': 'GCCcore', 'version': '12.3.0'}
+
+builddependencies = [('binutils', '2.40')]
+
+dependencies = [('Python', '3.11.3'),
+                 ('Python-bundle-PyPI', '2023.06'),  # It checks for it after installed
+                 ]
```

The CI check is not green because these two code style errors:
```
pytest-rerunfailures-12.0-GCCcore-12.3.0.eb:14:18: E127 continuation line over-indented for visual indent
pytest-rerunfailures-12.0-GCCcore-12.3.0.eb:15:18: E124 closing bracket does not match visual indentation
```
```suggestion
dependencies = [
    ('Python', '3.11.3'),
    ('Python-bundle-PyPI', '2023.06'),  # It checks for it after installed
]
```